expressEngine/product/service/product.js

```diff
@@ -0,0 +1,40 @@
+// import service libraries
+import { BaseService, DbService } from '../../lib/service/index';
```

Comment:
replace DbService to DBService

Reply:
DbService is correct notation. It doesn't need to be changed